src/fmeval/data_loaders/util.py

```diff
@@ -180,8 +180,13 @@ def _is_valid_s3_uri(uri: str) -> bool:
     if parsed_url.scheme.lower() not in ["s3", "s3n", "s3a"]:
         return False
     try:
+        s3_client = (
+            boto3.client("s3", region_name=BUILT_IN_DATASET_DEFAULT_REGION)
```

Comment:
Is this the only place it will fail?

Reply:
Updated the PR with a util method called every time a boto3 s3 client is created.